sway/handlers.c

```diff
@@ -344,13 +349,13 @@ static void handle_view_geometry_request(wlc_handle handle, const struct wlc_geo
 		view->desired_width = geometry->size.w;
 		view->desired_height = geometry->size.h;
 
-		if (view->is_floating) {
-			view->width = view->desired_width;
-			view->height = view->desired_height;
-			view->x = geometry->origin.x;
-			view->y = geometry->origin.y;
-			arrange_windows(view->parent, -1, -1);
-		}
+		/* if (view->is_floating) { */
+		/* 	view->width = view->desired_width; */
+		/* 	view->height = view->desired_height; */
+		/* 	view->x = geometry->origin.x; */
+		/* 	view->y = geometry->origin.y; */
+		/* /1* 	arrange_windows(view->parent, -1, -1); *1/ */
+		/* } */
```

Comment:
Sorry, the config I linked on IRC is actually a different one than I was testing with at the time.

    for_window [title="^gnome-calculator$"] floating enable

It freaks out by appearing briefly and then becoming very small, I think. I can still move up to the floating layer but I don't see it and I can't interact with it. I can also close it (with keybindings).